src/mysqlsh_helpers.py

```diff
@@ -278,24 +278,38 @@ def add_instance_to_cluster(self, instance_address) -> None:
         """
         options = {
             "password": self.cluster_admin_password,
-            "recoveryMethod": "clone",
+            "recoveryMethod": "auto",
```

Comment:
I guess this can be removed as it's being overriden on L291.

Reply:
good catch. removed in 27779e4d85e1bc2ab3ce6b2d5b27e9078fd09b99